src/UKHO.S100PermitService.API/Program.cs

```diff
@@ -114,8 +113,9 @@ private static void ConfigureServices(WebApplicationBuilder builder)
             builder.Services.AddDistributedMemoryCache();
 
             builder.Services.Configure<EventHubLoggingConfiguration>(builder.Configuration.GetSection("EventHubLoggingConfiguration"));
-            builder.Services.Configure<HoldingsServiceApiConfiguration>(configuration.GetSection(HoldingsServiceApiConfiguration));
-            builder.Services.Configure<UserPermitServiceApiConfiguration>(configuration.GetSection(UserPermitServiceApiConfiguration));
+            builder.Services.Configure<HoldingsServiceApiConfiguration>(configuration.GetSection("HoldingsServiceApiConfiguration"));
+            builder.Services.Configure<UserPermitServiceApiConfiguration>(configuration.GetSection("UserPermitServiceApiConfiguration"));
+            builder.Services.Configure<ProductKeyServiceApiConfiguration>(builder.Configuration.GetSection("ProductKeyServiceApiConfiguration"));
```

Comment:
To maintan consistent can we configure as
```suggestion
            builder.Services.Configure<ProductKeyServiceApiConfiguration>(configuration.GetSection("ProductKeyServiceApiConfiguration"));
```

Reply:
Resolved